tests/Http/PostTest.php

```diff
@@ -762,6 +762,43 @@ public function testPostsIndexAsNonAdminNonOwner()
         ]);
     }
 
+    /**
+     * Test for retrieving all posts with some anonymous posts.
+     * Non-owners should not northstar_id or location.
+     *
+     * GET /api/v3/posts
+     * @return void
+     */
+    public function testPostsIndexWithAnonymousPosts()
+    {
+        // Create an accepted post.
+        $regularPost = factory(Post::class, 'accepted')->create();
+
+        // Create 1 post that is anonymous.
+        sleep(1);
```

Comment:
Why is this `sleep(1)` needed here?

Reply:
@DFurnes if `sleep(1)` isn't used, both the `$regularPost` and `$anonymousPost` are created at the same time bc the test goes so fast! I want to deliberately make sure the `$anonymousPost` is created after the `$regularPost` to make sure that when we hit the index, the `$anonymousPost` is the first in the array since it returns records in descending order.

Reply:
It _should_ still fall back to ID for ordering if two posts have the same `created_at`, I think. If ou remove this, do the tests become flaky?

We want to avoid using `sleep()` in our test suite whenever possible, since if we over-use it it'd significantly slow things down. (We have 137 tests that run in ~20 seconds now... imagine if each of those had a `sleep(1)` in them! 😨)

Reply:
@DFurnes yeah I've tried to remove this and then the tests fail :-/

Reply:
Okay, bummer! We can use the `mockTime` method in that case ([example](https://github.com/DoSomething/rogue/blob/1cf8256dcc6426e8b8f8ee4205e6d3dd740a0584/tests/Unit/Models/ReactionModelTest.php#L20-L28)).

Reply:
oh right I forgot about that! Updated! :)